src/helpers/secret_helper.ts

```diff
@@ -0,0 +1,35 @@
+import {S3 } from 'aws-sdk';
+
+const bucket = '268644478934-miniconomy-creds';
```

Comment:
Would it maybe be better to make this a secret?
Whats stopping someone from grabbing our certs?